### PR TITLE
Personal Portfolio Website

### PR DESCRIPTION
**Pull request by ravi0900**
### What PR has:-
* I have created a personal portfolio website guided project, and added all step and instruction description.
* I have already created a starter template repository for this project and added cdmrc file.
* I tried my best, suggestion and change request are welcomed.
* contribution for Codedamn ❤️, hacktoberfest22 ❤️.

### CODE REVIEW

**Comment by ravi0900** [comment]
😥 sad, i was following this PR from the day 1, now allowed edit by maintainer, you can make change request, closing request hurts !!

**Comment by Pranavtechie** [comment]
Hey @ravi0900 , the project submissions are globally visible in https://codedamn.com/projects and these guided projects are meant for other learners to build them on their own. Creating a Portfolio repo with the a name was not a good idea. 

The project design could have been much better. We already have a few good portfolio projects in the project section already like the Astral Portfolio Page, which does not focus any name or person

**Comment by ravi0900** [comment]
> Hey @ravi0900 , the project submissions are globally visible in https://codedamn.com/projects and these guided projects are meant for other learners to build them on their own. Creating a Portfolio repo with the a name was not a good idea.
> 
> The project design could have been much better. We already have a few good portfolio projects in the project section already like the Astral Portfolio Page, which does not focus any name or person



> Hey @ravi0900 , the project submissions are globally visible in https://codedamn.com/projects and these guided projects are meant for other learners to build them on their own. Creating a Portfolio repo with the a name was not a good idea.
> 
> The project design could have been much better. We already have a few good portfolio projects in the project section already like the Astral Portfolio Page, which does not focus any name or person

ok I can understand that this is unprofessional behaviour, sorry for mentioning that, I will change those design template image make better readme and attach nicer images, now I'm improving more, My project is really helpful for learning and creating portfolio website with html css and javascript in better way, and somewhat better than astral portfolio website, I'm fixing those problem and committed to make a better PR, I'm not that good, so I spend a lot of time making this and still ready to make it better as you say, I was already got rejected for one issue, I was following this for so long time, please accept my project PR It will really motivate me to make more efforts with happy mind.

**Comment by Pranavtechie** [comment]
Hey @ravi0900, I understand your situation. Do make your next PR when you're ready. Hacktoberfest responses have already been closed so we can't consider your PR as part of it. However, we can still merge your PR.

**Comment by ravi0900** [comment]
> Hey @ravi0900, I understand your situation. Do make your next PR when you're ready. Hacktoberfest responses have already been closed so we can't consider your PR as part of it. However, we can still merge your PR.

You haven't comment a single time that you need this change so that I was assuming all is fine if not perfect at least, Please review my starter file again, reopen this PR, I think I have made the changes as per your expectation.
see this starter repo:- https://github.com/ravi0900/portfolio-website-codedamn
I hope you like this one, it is a medium level project and I looking forward to make my contribution.

**Comment by Pranavtechie** [comment]
@ravi0900 Do you have any other projects that you have built apart from the portfolio website page? The essence of building projects is in building the actual technical implementation of an idea, we already have a lot of projects that help users in understanding and implement basic User Interfaces. I do understand the need for having a portfolio website, but that does not  give the best

**Comment by ravi0900** [comment]
> @ravi0900 Do you have any other projects that you have built apart from the portfolio website page? The essence of building projects is in building the actual technical implementation of an idea, we already have a lot of projects that help users in understanding and implement basic User Interfaces. I do understand the need for having a portfolio website, but that does not mean bloating the projects section with more portfolio websites is a good idea.

I also added demo url https://portfolio-website-dfsgm.netlify.app, have you checked that carefully ? the project is perfect for practicing good css and javascript and after completing the project user will have some useful code for them, and I'm hoping that this will not be a bloat project, I also checked that astral and some other project carefully that how they evolved and build, and I made the changes that are required and I feel I made a nice addition to https://codedamn.com/projects, and also you just assign me what I chosen for contribution in hacktoberfest and I worked on #114 so now talking about other project is pointless for me in this PR, may be in future I'll try to contribute again in this Repo, I think we both have done with this conversation, so let's have a final call, if you consider this as good guided project you can add this or it just fine to close this PR. I'll move on. thanks !

**Comment by ravi0900** [comment]
thank you for reopening this request 😍❤️.

**Comment by ravi0900** [comment]
The branch has been updated.

File: portfolio-website/spec.yml [hunk 1]
```diff
@@ -0,0 +1,36 @@
+name: portfolio-website-guided-project
```
[the remark this answers is not in the document]
Comment (Pranavtechie) [reply portfolio-website/spec.yml:1]: @ravi0900 , you still haven't update the project name

File: portfolio-website/spec.yml [hunk 2]
```diff
@@ -0,0 +1,36 @@
...
+tags: ["HTML/CSS","JavaScript"]
+starter-files: https://github.com/ravi0900/portfolio-website-codedamn
+type: frontend
+level: medium
+cover-image: https://raw.githubusercontent.com/ravi0900/portfolio-website-codedamn/master/assets/design/Preview1.gif
```
Comment (Pranavtechie) [inline portfolio-website/spec.yml:6]: Please insert a `.png` image here

File: portfolio-website/step0/Instructions.md [hunk 3]
```diff
@@ -0,0 +1,24 @@
+In this step, our goal is to create the index.html with HTML tag elements and complete the basic Markup. 
+
+You have to add all the HTML tags.
+create li tag and add  
+
+- `Home`, `About me`, `Project` H1 heading 
+- Description 
+
+    add h1 tag for li tag items 
+    ```code
```
Comment (Pranavtechie) [inline portfolio-website/step0/Instructions.md:10]: use `html` instead of `code` for correct color hightlighting

**Review by Pranavtechie — CHANGES_REQUESTED** [review]
Please update the the codeblock names that you have set to available language names for proper syntax highlighting. 

use `html` , `css` and `js` 

Refer to this article if you need to understand this https://www.freecodecamp.org/news/how-to-format-code-in-markdown/

**Comment by ravi0900** [comment]
so I have update the branch with proper code highlights, project name and preview image, preview image pixel size is high, say if need in less Height x Width.

**Comment by ravi0900** [comment]
I have updated again and fixed some typos and content improvement.

**Review by Pranavtechie — APPROVED** [review]

**Comment by Pranavtechie** [comment]
@ravi0900 Share your email address here or reach out to me at pranav@codedamn.com

**Comment by ravi0900** [comment]
> @ravi0900 Share your email address here or reach out to me at [pranav@codedamn.com](mailto:pranav@codedamn.com)

I'm extremely thankful to you for merging this PR ❤️, my email id is `kushwaharaviraj8@gmail.com `

**Comment by ravi0900** [comment]
> @ravi0900 Share your email address here or reach out to me at [pranav@codedamn.com](mailto:pranav@codedamn.com)

although project is not guide because I forgot to include the guide true flag in specs.yml, I can push update ! should I ?

**Pull request merged**